### ISSUE TITLE
How do we get md files to open as notebooks by default in jupter notebook/lab?

### ISSUE THREAD

**Issue by tomdonaldson** [NASA-NAVO/navo-workshop#138]

**Comment by bsipocz** [comment]
For reference, I have these installed locally where it works out of the box:

```
$ ~/munka/IPAC/navo-workshop [hacking|⚑ 5] $ pipl jupyter
jupyter                           1.0.0
jupyter-cache                     0.5.0
jupyter_client                    7.4.7
jupyter-console                   6.4.4
jupyter-contrib-core              0.4.2
jupyter_core                      5.1.0
jupyter-nbextensions-configurator 0.6.1
jupyter-server                    1.23.3
jupyterlab-pygments               0.2.2
jupyterlab-widgets                3.0.3

[notice] A new release of pip available: 22.2.2 -> 23.3.1
[notice] To update, run: pip install --upgrade pip
$ ~/munka/IPAC/navo-workshop [hacking|⚑ 5] $ pipl notebook
notebook                          6.5.2
notebook_shim                     0.2.2
```

**Comment by stargaser** [comment]
@zoghbi-a found out how to change the settings to make a double-click work. The steps are listed in #143.
